kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/ZooKeeperHiveClientHelper.java

```diff
@@ -112,7 +115,7 @@ static void configureConnParams(JdbcConnectionParams connParams)
       updateParamsWithZKServerNode(connParams, zooKeeperClient, serverNode);
     } catch (Exception e) {
       throw new ZooKeeperHiveClientException(
-          "Unable to read HiveServer2 configs from ZooKeeper", e);
+          "Unable to read HiveServer2 configs from ZooKeeper: " + e.getMessage());
```

Comment:
why change it? we should propagate the inner exception otherwise we will lost the stacktrace